packages/api/tests/api/admin/me/get.test.ts

```diff
@@ -0,0 +1,23 @@
+import { get } from '../../../../src/api/admin/me/get';
+
+const test = {
+  user: '1',
+};
+
+const mockRequest: any = {
+  user: { id: '1' },
+  admin: test,
+};
+
+const mockResponse: any = {
+  send: jest.fn(),
+};
+
+describe('admin me handler', () => {
+  it('returns the admin associated with the request', () => {
+    // test
+    get(mockRequest, mockResponse);
+    // assert
```

Comment:
I believe these were just copy and pasted; let's get rid of them unless this was an intentional addition
```suggestion
    get(mockRequest, mockResponse);
```